lib/public_key/asn1/OTP-PKIX.asn1

```diff
@@ -234,8 +234,8 @@ countryName ATTRIBUTE-TYPE-AND-VALUE-CLASS ::= {
  -- we define and use the type below instead of X520countryName
 
```

Comment:
We suggest to add the following to the comment:

 -- We accept utf8String encoding of the US-ASCII
 -- country name code and the mix up with other country code systems
 -- tha uses three characters instead of two.

Reply:
done